cmd/boots/dhcp.go

```diff
@@ -83,7 +83,7 @@ func (d dhcpHandler) serveDHCP(w dhcp4.ReplyWriter, req *dhcp4.Packet) {
 
 	circuitID, err := getCircuitID(req)
 	if err != nil {
-		mainlog.With("mac", mac, "err", err).Info("error parsing option82")
+		mainlog.With("mac", mac).Error(err, "error parsing option82")
```

Comment:
lets just get rid of this whole branch here. Don't want to treat this as an error and no real point in logging this imo.

Reply:
I don't know enough about this to feel comfortable changing the logic, I really just tried to standardize how the error logging worked as I debugged issues

Reply:
sounds fine.